### PR TITLE
add task

### PR DESCRIPTION
**Pull request by Ren6436**
- [DEMO LINK](https://Ren6436.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://Ren6436.github.io/layout_product-cards/report/html_report/)

Unfortunately, I have run out of ideas on how to align the stars and the review text, so I would appreciate any comments. Thank you.

### CODE REVIEW

File: src/styles/blocks/card.scss [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      line-height: 18px;
+    }
+  }
+
+  &__button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 16px;
+    background-color: #00acdc;
```
Comment (anastasiiavorobiova) [inline src/styles/blocks/card.scss:51]: Consider using variables for repeating colors

File: src/styles/blocks/card.scss [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    color: #fff;
```
Comment (anastasiiavorobiova) [inline src/styles/blocks/card.scss:52]: Consider using variables for repeating colors

File: src/styles/blocks/card.scss [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    border-radius: 5px;
+    border: none;
+    text-decoration: none;
+    width: 166px;
```
Comment (anastasiiavorobiova) [inline src/styles/blocks/card.scss:59]:
```suggestion
    width: 100%;
```

File: src/styles/blocks/card.scss [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+      background-color: transparent;
+      color: #00acdc;
+      font-weight: 700;
+      font-size: 14px;
+      line-height: 16px;
+      border-radius: 5px;
+      border: 1px solid #00acdc;
+      text-decoration: none;
+      width: 166px;
+      height: 40px;
```
Comment (anastasiiavorobiova) [inline src/styles/blocks/card.scss:76]:
```suggestion
      background-color: transparent;
      color: #00acdc;
```

File: src/styles/blocks/stars.scss [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    height: 16px;
+    background-position: 50%;
+    background-repeat: no-repeat;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+
+  &--4 {
```
Comment (anastasiiavorobiova) [inline src/styles/blocks/stars.scss:33]: Consider using styles from the previous task to keep modifiers working

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Feel free to ask for help in the chat to fix tests

**Review by vadiimvooo — CHANGES_REQUESTED** [review]
Seems like you have not deployed your changes. Please feel free to ask for help in fe_chat.

**Comment by Ren6436** [comment]
I had some issues with this branch, so I made a new pull request. Sorry about that.
You can find the link to the new pull request below.

https://github.com/mate-academy/layout_product-cards/pull/4592

> Seems like you have not deployed your changes. Please feel free to ask for help in fe_chat.

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Pls fix prev comments
Pls ask for help in **fe_chat** if you don't understand something

**Pull request closed without merging**